embed-code-go/fragmentation/encoding.go

```diff
@@ -49,7 +49,8 @@ func IsEncodedAsText(filePath string) bool {
 	// Read the entire file into memory.
 	content, err := os.ReadFile(filePath)
 	if err != nil {
-		panic(err)
+		// If error occurs, it's not a text-encoded file.
```

Comment:
Should we do something about it? Or we consider it normal?

I would say our users deserve to know their input is obviously wrong. And to me that would be a show-stopper.